ui/src/shared/components/selectorList/SelectorListBody.tsx

```diff
@@ -0,0 +1,51 @@
+// Libraries
+import React, {PureComponent, ReactNode} from 'react'
+
+// Components
+import {DapperScrollbars} from '@influxdata/clockface'
+
+interface Props {
+  scrollable: boolean
+  addPadding: boolean
+  testID: string
+}
+
+export default class BuilderCardBody extends PureComponent<Props> {
```

Comment:
Should `BuilderCardBody` be `SelectorListBody`?

Reply:
I updated it